src/AppBundle/Controller/RfXVoteCalculatorController.php

```diff
@@ -0,0 +1,228 @@
+<?php
+/**
+ * This file contains the code that powers the RfX Vote Calculator page of xTools.
+ *
+ * @category RfXVoteCalculator
+ * @package  AppBundle\Controller
+ * @author   XTools Team <xtools@lists.wikimedia.org>
+ * @license  GPL 3.0
+ * @link     http://xtools.wmflabs.org/rfap
+ */
+
+namespace AppBundle\Controller;
+
+use Symfony\Bundle\FrameworkBundle\Controller\Controller;
+use Symfony\Component\CssSelector\Exception\InternalErrorException;
+use Symfony\Component\Debug\Exception\ContextErrorException;
+use Symfony\Component\HttpFoundation\Request;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
+use Xtools\ProjectRepository;
+use Xtools\RFA;
+use Xtools\User;
+
+// Note: In the legacy XTools, this tool was referred to as "rfap."
+// Thus we have several references to it below, including in routes
+
+/**
+ * Class RfXVoteCalculatorController
+ *
+ * @category RfXVoteCalculator
+ * @package  AppBundle\Controller
+ * @author   XTools Team <xtools@lists.wikimedia.org>
+ * @license  GPL 3.0
+ * @link     http://xtools.wmflabs.org/rfap
+ */
+class RfXVoteCalculatorController extends Controller
+{
+
+    /**
+     * Get the tool's shortname.
+     *
+     * @return string
+     */
+    public function getToolShortname()
+    {
+        return 'rfap';
+    }
+
+    /**
+     * Renders the index page for RfXVoteCalculator
+     *
+     * @Route("/rfap", name="rfap")
+     * @Route("/rfap", name="RfXVoteCalculator")
+     *
+     * @return \Symfony\Component\HttpFoundation\Response]
```

Comment:
extra `]`? I think we're just using `Response` in our other controllers, not that it matters